regression/goto-analyzer/approx-array-variable-const-fp/test.desc

```diff
@@ -1,7 +1,7 @@
 CORE
 main.c
 --show-goto-functions --pointer-check
-^Removing function pointers and virtual functions$
+^Removal of function pointers and virtual functions$
```

Comment:
Perhaps put that change of the status message in a commit of its own so that the biggest change in this series does come without any observable impact on tests?

Reply:
Resolved